charts/datahub/subcharts/datahub-frontend/values.yaml

```diff
@@ -49,6 +49,9 @@ service:
   annotations: {}
     # service.beta.kubernetes.io/aws-load-balancer-internal: "true"
 
+serviceMonitor:
+  create: false
```

Comment:
Shouldn't we create service monitors by default since they only get activated if and only if `global.datahub.monitoring.enablePrometheus` is also enabled?

Reply:
if they don't have serviceMonitor CRD, the deployment would fail. It's an advanced feature, if somebody know what they are doing, then they can enable.